websites/K/Kingdom of Loathing/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+        "name": "gausie",
+        "id": "263438077865754644"
+    },
+    "service": "Kingdom of Loathing",
+    "altnames": ["KoL", "KoLmafia"],
+    "description": {
+        "en": "Kingdom of Loathing (abbreviated KoL) is a browser-based multiplayer role-playing game designed and operated by Asymmetric Publications"
+    },
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/ZRf8Giq.png",
+    "thumbnail": "https://i.imgur.com/ZhJPGTN.png",
+    "color": "#0000ff",
+    "tags": ["rpg", "asym", "kol"],
+    "category": "games",
+    "url": "kingdomofloathing.com",
+    "regExp": "([a-z0-9]+\\.)*kingdomofloathing.com|(127\\.0\\.0\\.1|localhost):6008\\d",
```

Comment:
why localhost tho? xD

Reply:
Because another extremely common way to access Kingdom of Loathing is via a tool called KoLmafia, which opens a port on 6008{1-9} on your machine. I expect more than half the users to be accessing it from there!